assets/javascripts/discourse/controllers/admin-plugins-slack.js.es6

```diff
@@ -4,7 +4,7 @@ import { popupAjaxError } from 'discourse/lib/ajax-error';
 
 export default Ember.Controller.extend({
   categories: function() {
-    return [Discourse.Category.create({ name: 'All Categories', id: 0, slug: '*'})].concat(Discourse.Category.list());
+    return [Discourse.Category.create({ name: 'Choose a Category', id: -1, slug: null}), Discourse.Category.create({ name: 'All Categories', id: 0, slug: '*'})].concat(Discourse.Category.list());
```

Comment:
Style nitpick: The line is too long here which makes it hard to read without having to scroll.

```
[
 1,
 2,
 3
]
```

Something like this will be better